src/benchmark/operators/sql_benchmark.cpp

```diff
@@ -26,9 +26,11 @@ class SQLBenchmark : public BenchmarkBasicFixture {
     // Add tables to StorageManager.
     // This is required for the translator to get the column names of a table.
     auto& storage_manager = StorageManager::get();
-    storage_manager.add_table("customer", load_table("src/test/tables/tpch/minimal/customer.tbl", Chunk::MAX_SIZE));
-    storage_manager.add_table("lineitem", load_table("src/test/tables/tpch/minimal/lineitem.tbl", Chunk::MAX_SIZE));
-    storage_manager.add_table("orders", load_table("src/test/tables/tpch/minimal/orders.tbl", Chunk::MAX_SIZE));
+    storage_manager.add_table("customer",
+                              load_table("src/test/tables/tpch/tpch-minimal/customer.tbl", Chunk::MAX_SIZE));
+    storage_manager.add_table("lineitem",
+                              load_table("src/test/tables/tpch/tpch-minimal/lineitem.tbl", Chunk::MAX_SIZE));
+    storage_manager.add_table("orders", load_table("src/test/tables/tpch/tpch-minimal/orders.tbl", Chunk::MAX_SIZE));
```

Comment:
What's wrong with `tpch/minimal`?

Reply:
wanted the scale factor on the folders name. Do you prefer `tpch/minimal` and `tpch/sf-0001` over the current structure?

Reply:
But... the SF is not part of the folder name.

I would go with `tpch/scale-1`.

Reply:
Or is it 0,001? In that case, I would call the folder `tpch/scale-0.001`. With your version, I thought it was padding.